src/Components/Api.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios"
+
+const api = axios.create({
+    baseURL: "https://sabrinas-nc-news.onrender.com/api/",
+})
+
+const getTopics = () => {
+    return api.get("/topics").then(({ data }) => {
+        return data.topics
+      });
+}
+
+const fetchArticlesByTopics = (selectedTopic) => {
+    if (selectedTopic === "all") {
+        return api.get("/articles").then(({ data }) => { 
+          return data.articles
+        })
+      } else {
+        return api.get(`/articles?topic=${selectedTopic}`).then(({ data }) => {
+          return data.articles
+        })
+      }
+}
```

Comment:
Axios can be used for performing optional queries if you pass an object containing the query to the request, this would clean up the `if` and `else` 
See [docs](l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=optional-queries)